include/cutlass/gemm/collective/xe_mma.hpp

```diff
@@ -100,8 +100,36 @@ struct CollectiveMma<MainloopIntelXeXMX16<Stages, Schedule>, TileShape_, Element
   static constexpr auto Num_SGs = ATOM_N * ATOM_M * ATOM_K;
   static constexpr uint32_t MaxThreadsPerBlock = size(TiledMma{});
 
-  using Copy_A = typename Copy_Traits<GmemTiledCopyA, StrideA>::template DefaultTiledCopy<ElementA>;
-  using Copy_B = typename Copy_Traits<GmemTiledCopyB, StrideB>::template DefaultTiledCopy<ElementB>;
+  // Helper struct to deduce CopyOpA type
+  template<class EA, class SA, class TM, class GTCA>
```

Comment:
Please write out the type names in detail:
```suggestion
  template<class ElementA, class StrideA, class TiledMMA, class GmemTiledCopyA>
```